.github/workflows/bedevere.yml

```diff
@@ -0,0 +1,48 @@
+name: Bedevere
+
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - synchronize
+      - reopened
+      - edited
```

Comment:
A repetition of the first list entry.

```suggestion
```

Reply:
As an alternative, the first entry may be deleted instead.

Reply:
Thanks, yes I've removed the duplicated event, and added empty lines at the end of file.